kotlinx-coroutines-core/jvm/src/Executors.kt

```diff
@@ -38,13 +38,23 @@ public abstract class ExecutorCoroutineDispatcher: CoroutineDispatcher(), Closea
 
 /**
  * Converts an instance of [ExecutorService] to an implementation of [ExecutorCoroutineDispatcher].
+ *
+ * Note, that if the underlying executor uses limited queues and throws [RejectedExecutionException] on
```

Comment:
Job is cancelled iff the executor refects a task, indipendently if is queue is limited or not (ie on shutdown). Same in `Executor.asCoroutineDispatcher()`.

Reply:
I've clarified docs a bit. Check it out.

Reply:
Looks good to me.